packages/@okta/vuepress-site/docs/guides/style-the-widget/customization-examples/index.md

```diff
@@ -55,7 +61,7 @@ To access the application's client ID (which uniquely identifies the application
 <script>
   function getClientId() {
     if (!OktaUtil) return undefined;
-    
+
```

Comment:
Can poeple find out the unique IDs of their apps in a particular place in the Apps screen of Admin Console?

Reply:
good suggestion. thank you.